### DISCUSSION TITLE
Welcome to FaceAPI discussions!

### DISCUSSION THREAD

**Discussion by vladmandic** [vladmandic/face-api#22] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vladmandic** [comment]
I'll fix it today. 
But please when there is a problem, create a new issue - do NOT just append to existing discussion.
Especially like this - how is your post related to "Welcome to FaceAPI discussions" topic?!

**Reply by qqw1943** [reply]
sorry for what i did.comment has been deleted.it won't happen again

**Reply by vladmandic** [reply]
anyhow, problem has been resolved

actual stable version of faceapi was not affected,
only current development version which is not published on npm, but it is what you see live on github
